<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupStatus</class>
 <widget class="QWidget" name="GroupStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TabBar" name="tabBar" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_group">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="TableView" name="tableView_group"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_channel">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="TableView" name="tableView_channel"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabBar</class>
   <extends>QWidget</extends>
   <header>tabbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
